timing_script.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/tcsh
+
+echo HOST:"\t"$HOST
+
+echo PARAMS:"\t"$1"\t"$2"\t"$3
+
+echo VERSION:"\t"`git describe --tags`
+
+echo Loading Topographica ant quitting:
+
+time ./topographica -c quit
+
+echo ----------------------------------
+
+echo Loading Topographica + setting up gcal_oo_or.ty using $1 cortex density + running 0 iterations \(gpu=$3\):
+
+time ./topographica -p cortex_density=$1 -p gpu=$3 examples/gcal_oo_or.ty -c "topo.sim.run(0)"
+
+echo ----------------------------------
+
+echo Loading Topographica + setting up gcal_oo_or.ty using $1 cortex density + running $2 iterations \(gpu=$3\):
+
+time ./topographica -p cortex_density=$1 -p gpu=$3 examples/gcal_oo_or.ty -c "topo.sim.run($2)"
+
+echo ----------------------------------
```

Comment:
It would be good to have a runnable notebook, though, both as an example of benchmarking in general and to allow people to benchmark the GPU code.  Maybe it can be merged into Benchmarking.ipynb?  Note that I haven't been able to actually see Benchmarking.ipynb run yet.



-- 
The University of Edinburgh is a charitable body, registered in
Scotland, with registration number SC005336.